wordpress/web/app/themes/bioco/assets/app.css

```
@@ -127,14 +127,41 @@ header:has(.bioco-navigation-shell) {
 }
 
 .bioco-primary-nav .bioco-primary-cta {
+  --bioco-nav-cta-background: var(--wp--preset--color--bioco-green);
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)